fitsio/test.py

```diff
@@ -1062,14 +1062,26 @@ def testReadFlipAxisSlice(self):
                         "Data are not the same (Expected shape: {}, actual shape: {}.".format(
                             expected_data.shape, rdata.shape))
 
-                rdata = hdu[:,130:70:6]
+                rdata = hdu[:,130:70:-6]
 
                 # Expanded by two to emulate adding one to the start value, and adding one to the calculated dimension.
                 expected_data = data[:,130:70:-6]
 
                 numpy.testing.assert_array_equal(expected_data, rdata,
                         "Data are not the same (Expected shape: {}, actual shape: {}.".format(
                             expected_data.shape, rdata.shape))
+
+                try:
+                    hdu[:,90:60:4]  # Positive step integer does not follow numpy syntax.
```

Comment:
What does this comment mean, that it does not follow numpy syntax?

Reply:
with a numpy array, the above would return an empty array

Reply:
That's correct, it does return an empty array.  `numpy` syntax wants a negative number when `x` < `y` for a slice.  Given that it will always return an empty array in this situation, I thought it better to inform the user.

Reply:
But are you not raising an error for that case?

Reply:
Right, by inform I mean it won't return what you think it will return, so that may be reason enough to die.  

In `CFITSIO`, the syntax is to use a positive integer for the `step` value for this use case, but with the `numpy` syntax, it requires a negative one, so there may be confusion from the caller's standpoint.

Reply:
I think maybe the right thing to do would be to return the empty array rather than raising an exception.  We have the type info available, so we could do that.

If you want we could do it in a different PR.  Does anyone else have an opinion on it?

Reply:
Sure.  Lines 280 to 283 in `image.py` would perhaps instead look like:
```python
            if (step > 0 and stop < start) or (step < 0 and start < stop):
                return numpy.empty(0, dtype=npy_dtype)
```

Then the tests could just check for an empty array.